### DISCUSSION TITLE
Get backref data

### DISCUSSION THREAD

**Discussion by Master-Y0da** [pydantic/pydantic#2643] [Question]
I have the following models(using tortoise-orm):

`

    class Auth_Rol_Permiso(models.Model, TimestampMixin):

    id = fields.UUIDField(pk=True)
    permiso = fields.ForeignKeyField('models.Permiso', related_name="auth_permiso", on_delete=fields.CASCADE) 
    user = fields.ForeignKeyField("models.UserModel", related_name="auth_user", on_delete=fields.CASCADE) // User model

    class Permiso(models.Model):

    id = fields.UUIDField(pk=True)
    nombre = fields.CharField(max_length=100)

With the next pydantic models:

`

    class RolPost(BaseModel):
        nombre: str 

    class UserPermission(BaseModel):
       permiso: RolPost

    class User(BaseModel):
     nombre: str
     auth_user: List[UserPermission] --> If I set auth_user only as a List, it works and returns fields from Auth_Rol_Permiso
                                      If I assign List[UserPermission] gives me error auth_user -> 0 -> permiso  field required (type=value_error.missing)` 

Basically I want to get User info and through backref **auth_user**, get a response like this:
`
 (including nombre from Permiso class, actually it just returns until auth_user array level)
   
    UserInfo:
     {
      "nombre" : 'Master Yoda'
      "auth_user" : [ {  "permiso": { "nombre": "Algo" }, "id": 1212 }, {  "permiso": { "nombre": "Algo2" }, "id": 1213 } ] 
      }`

Please give me a hand with this...Thanks for your help.

**Comment by PrettyWood** [comment]
Hi @Ivar 
Please can you format properly your question and be more explicit? It's really hard to read and understand

**Reply by Master-Y0da** [reply]
Hi @PrettyWood 
I delve into the question here [fastapi-users-pydantic](https://github.com/frankie567/fastapi-users/issues/600), since I think it's a bug.
Feel free to contrib. Thanks for your time.
